Applications/Spire/Source/KeyBindings/CancelKeyBindingsModel.cpp

```diff
@@ -0,0 +1,51 @@
+#include "Spire/KeyBindings/CancelKeyBindingsModel.hpp"
+#include "Spire/Spire/ValidatedValueModel.hpp"
+
+using namespace boost;
+using namespace Spire;
+
+CancelKeyBindingsModel::CancelKeyBindingsModel() {
+  for(auto i = 0; i < 13; ++i) {
+    auto operation = static_cast<Operation>(i);
+    m_bindings[operation] = make_validated_value_model<QKeySequence>(
```

Comment:
An ```unordered_map``` is not appropriate when indexed by an incrementally increasing integer. An array is the optimal data structure in such a scenario.